app/src/main/java/com/github/versus/offline/SimpleTestPost.java

```diff
@@ -12,25 +12,64 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * A class designed for testing the Cache manager
+ */
 public final class SimpleTestPost extends Post {
-
+    /**
+     * Creates an instance of the simple post with a default title
```

Comment:
some of the comments just say something which can be inferred from the method name

Reply:
Doesn't harm though :-)